processor/src/main/java/org/mapstruct/ap/internal/processor/MapperCreationProcessor.java

```diff
@@ -441,6 +444,23 @@ else if ( method.isStreamMapping() ) {
         return mappingMethods;
     }
 
+    private Javadoc getJavadoc(TypeElement element) {
+        JavadocGem javadocGem = JavadocGem.instanceOn( element );
+
+        if ( javadocGem == null ) {
+            return null;
+        }
+
+        Javadoc javadoc = new Javadoc.Builder()
```

Comment:
Should we perhaps have an error or warning if nothing is defined? It doesn't bring much value to use `@Javadoc` without anything

Reply:
That was a very good point indeed.

Reply:
Yes using the `FormattingMessenger` is the right approach for providing feedback. Have a look at something like `IterableMappingOptions#isConsistent` to see how we are doing it in other places